accounts/migrations/0001_squashed_0003_auto_20180203_2223.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.0.1 on 2018-02-03 22:31
```

Comment:
Just for curiosity, should migration files be added into .gitignore?

Reply:
They should be committed because the production machine can use them directly to migrate to the new schema. https://stackoverflow.com/a/28035246/5050657